src/index.html

```diff
@@ -7,12 +7,68 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="image">
+        <img
+          src="/src/images/imac.jpeg"
```

Comment:
The image path should be relative to the HTML file. Consider changing the path from '/src/images/imac.jpeg' to '../images/imac.jpeg' to ensure the image loads correctly.